.github/README.md

```diff
@@ -55,98 +55,98 @@ The DLx project recommends JSON because it has become the data interchange forma
 ```json
 {
   "transcription": {
-    "spa": "hola me llamo Daniel",
+    "sp": "hola me llamo Daniel",
```

Comment:
The language tag for Spanish is `es` or `spa`. (`es` is from ISO 639-1, and `spa` from 639-3, the latest version; both are still completely valid.) Let's use `es` (the official advice from IETF is to choose the shortest language tag possible).